pkgs/by-name/uc/ucblogo/package.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, wxGTK32
+, texinfo
+, tetex
+, wrapGAppsHook
+, autoconf-archive
+, autoreconfHook
+}:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation {
```

`rec` is not needed here